extensions/notification/utils/config.js

```diff
@@ -69,6 +69,14 @@ function filterConfigsByEvent(bnConfigs, event) {
  */
 function validateEntry(bnConfig, entry) {
     const { bucket, eventType } = entry;
+    /**
+     * if the event type is undefined, it is most likely an entry that is a
+     * placeholder for deletion or cleanup, these entries should be ignored and
+     * not be processed.
```

Comment:
* You say "is undefined" but use a falsey check below. Should this be `eventType === undefined`?
* "most likely" is not a very concrete description, it sounds like there is a chance it's something else. What that else is I have no idea.

Reply:
corrected.